test/hotspot/jtreg/compiler/arguments/TestCompilerCounts.java

```diff
@@ -27,6 +27,7 @@
  * @bug 8356000
  * @requires vm.flagless
  * @requires vm.bits == "64"
+ * @requires vm.debug
```

Comment:
Why you limited it to debug VM?

Reply:
Originally, I put this restriction there because `-XX:CodeCacheMinimumUseSpace` is a debug flag. But your comment prompted me to check if that flag has an ergonomic, which it does not. Hence, I made the test work for both debug and product in ab1c7d9.